roles/console/files/help/Control.rst

```diff
@@ -22,6 +22,45 @@ Displays a log of the commands used to start up the Admin Console and whether th
 
 Errors are displayed if the server is not available or the command server is not running or not ready.
 
+TailScaleControl
+----------------
+
+TailScale is a Virtual Private Network (VPN). It can be installed as part of IIAB, and you can manually install it on any of your other machines.
+
+Of course, to operate it requires an internet connection.
+
+### Do this First
+
+This panel only works if TailScale is installed. You can do so on the Configure menu.
+
+### How it works
+
+Once TailScale is installed, configured, and turned on, your IIAB server will become part of a private network with an IP address like 100.64.x.x
+
+Any machine on that network will be able to connect to any other over the internet, but other machines will not.
+
+Machines authenticate with a key generated by TailScale which is valid for one of three login URLs:
+
+* IIAB TailScale Login - This is the easiest because it is already setup; you just request a key. However, all IIAB servers are on the same network.
+* TailScale Default Login - If you or someone else creates a TailScale account and generates your own key, you can have your own TailScale network.
+* Custom Login URL - You can also run your own authentication server and generate your own keys. Most people will not want to do this.
+
+Configure the parameters below and click the button that will either say **Turn TailScale ON.** or **Turn TailScale OFF.**  depending on the state of TailScale, does what it says.
+
+### Paramters
```

Comment:
Just a few typos... to be fixed if possible?

- Parameters -> Parameters

- TailScale -> Tailscale